py3status/modules/xkb_layouts.py

```diff
@@ -174,7 +174,7 @@ class Py3status:
     button_next = 4
     button_prev = 5
     cache_timeout = 10
-    format = "Xkb [\?color=cyan {s}][\?color=darkcyan  {v}]"
+    format = "[\?color=cyan {s}][\?color=darkcyan {v}]"
```

Comment:
This omits a space. Try `setxkbmap -layout us,us -variant ,dvorak`. Also, the tests failed (non-identical formats).

Reply:
I fixed it, good catch

I also aligned colors with keyboard_layout

Reply:
I just realized something. You copying `thresholds` from keyboard_layouts would colorize only the known layouts. The other layouts not in thresholds (as well as the variants.... and other settings) wouldn't be colorized.

The basic `cyan` + `darkcyan` plays it safe and colorizes everything the same regardless.

Just want to share this small but potentially significant information. Cheers.